extensions/reactive-mssql-client/runtime/src/main/java/io/quarkus/reactive/mssql/client/runtime/MSSQLPoolRecorder.java

```diff
@@ -49,6 +49,8 @@
 @Recorder
 public class MSSQLPoolRecorder {
 
+    private static final boolean SUPPORTS_CACHE_PREPARED_STATEMENTS = false;
```

Comment:
In fact, when cached prepared statements are not supported, we shouldn't change anything.